app/classes/person.py

```diff
@@ -0,0 +1,318 @@
+import os
+from room import Room, Living_Space, Office, Living_Space
+from random import randint
+
+class Person(object):
+
+    """docstring for Person."""
+    people_list = {}
+    room_allocation_info = {}
+    person_info = []
+
+    def __init__(self):
+
+        pass
+
+    def add_person(self, first_name, second_name, person_job, employee_number, wants_room):
```

Comment:
refer to earlier comment in docstring and make `employee_number` be autogenerated by the system instead of user generating it.